pkg/splunk/enterprise/clustermaster.go

```diff
@@ -127,6 +128,12 @@ func ApplyClusterMaster(client splcommon.ControllerClient, cr *enterprisev1.Clus
 
 	// no need to requeue if everything is ready
 	if cr.Status.Phase == splcommon.PhaseReady {
+		//upgrade fron automated MC to MC CRD
+		namespacedName := types.NamespacedName{Namespace: cr.GetNamespace(), Name: GetSplunkStatefulsetName(SplunkMonitoringConsole, cr.GetNamespace())}
+		err = splctrl.VerfiyMCStatefulSetOwnerRef(client, cr, namespacedName)
+		if err != nil {
+			return result, err
```

Comment:
Should we really return an error here? May be just throw a warning & move on with other reconcile activities?

Same comment for other cases too